code/backend/Odin/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ dependencies {
     implementation("com.fasterxml.jackson.module:jackson-module-kotlin")
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlinx:kotlinx-datetime:$kotlinxDateTimeVersion")
+    implementation("org.jetbrains.kotlinx:kotlinx-serialization-json")
```

Comment:
I've tried finding the reference for this lib and couldn't find you using it anywhere.
Moreover, why'd you need another serialization library given SPRING already provides one for that.